src/analysis/inner_product_space/rayleigh.lean

```diff
@@ -219,7 +219,9 @@ end
 end complete_space
 
 section finite_dimensional
-variables [finite_dimensional 𝕜 E] [nontrivial E] {T : E →ₗ[𝕜] E}
+variables [finite_dimensional 𝕜 E] [_i : nontrivial E] {T : E →ₗ[𝕜] E}
+
+include _i
```

Comment:
How about dropping `[nontrivial E]` from variables and just adding it directly to `has_eigenvalue_supr_of_finite_dimensional` and `has_eigenvalue_infi_of_finite_dimensional`?

Reply:
This way `[nontrivial E]` appears before `T` in their arguments.  This is mostly cosmetic but I've seen other authors do such things to attain this goal.

Reply:
Feel free to leave it as you have it then. It's a minor subjective point.